### ISSUE TITLE
Documentation for `aria-hidden-focus` for single element is misleading and not fully correct

### ISSUE THREAD

**Issue by kospl** [dequelabs/axe-core#3252]
Product: axe-core

Expectation: `<input aria-hidden="true">` failing test for `aria-hidden-focus` and having clear help message explaining why this element is failing.

Actual: axe core 4.3.5 reports `aria-hidden-focus` violation with description about its elements (which don't exist).

Motivation: misleading help text made some tech support to think there is false positive with axe rule, because it's mentioning `do not contain elements`, and there are just one single DOM element, with no other elements that it contains that may be failing this test.
___

I know it sounds a little bit nerdy, but such help text have to be clear, as it's getting even worse when it's got to Lighthouse: https://github.com/GoogleChrome/lighthouse/blob/8a29854072202807ba54276beb9cdd9f244007b5/lighthouse-core/audits/accessibility/aria-hidden-focus.js#L18

__

minimal repro example:

    <input aria-hidden="true">

axe help text: "ARIA hidden element must not contain focusable elements"
Lighthouse title: "`[aria-hidden="true"]` elements do not contain focusable descendents"

**Comment by straker** [comment]
Thanks for the issue. We'll try to update the message to make it more clear.

**Comment by TechThomas** [comment]
Hey @straker how is the process going on improving the messaging. We actually just tweeted about this issue https://twitter.com/TechThomas/status/1488413276400922624

**Comment by straker** [comment]
We weren't able to get this into the 4.4.0 release which just went out, but we can try to get it added to 4.4.1.

**Comment by TechThomas** [comment]
@straker when will 4.4.1 be expected?

**Comment by straker** [comment]
Sorry, 4.4.1 was released not too long ago but did not include this fix (it was a quick release to fix a bug in 4.4.0). I'm not sure when we plan on releasing the next release, so I'm unable able to say when this will be addressed.

**Comment by padmavemulapati** [comment]
Validated with the latest axe-core, develop branch `axe.js` source, 
It was showing in 4.3.5 , for `aria-hidden-focus` rule failing description is ""Ensures aria-hidden elements do not contain focusable elements" , below is the screenshot

<img width="725" alt="image" src="https://user-images.githubusercontent.com/47409579/169813490-40e85eaf-da38-4f3e-9808-847b61a15123.png">

and now it got fixed and the description is appearing as "Ensures aria-hidden elements are not focusable nor contain focusable elements"
<img width="1471" alt="image" src="https://user-images.githubusercontent.com/47409579/169813909-d2798594-86e6-42ff-8955-9494a5ce2723.png">
Source validated to produce this violation is "<input aria-hidden="true">"
